docs/pages/api-reference/liveblocks-react-lexical.mdx

```diff
@@ -0,0 +1,223 @@
+
+const initialConfig = liveblocksConfig({
+  namespace: "MyEditor",
+  theme: {},
+  nodes: [],
+  onError: (err) => console.error(err),
+});
+
+function Editor() {
+  return (
+    <LexicalComposer initialConfig={initialConfig}>
+      <LiveblocksPlugin />
+      <RichTextPlugin
+        contentEditable={<ContentEditable />}
+        placeholder={<div>Enter some text...</div>}
+        ErrorBoundary={LexicalErrorBoundary}
+      />
+    </LexicalComposer>
+  );
+}
+```
+
+Learn more in our [get started guide](/docs/get-started/yjs-lexical-react).
+
+## liveblocksConfig
+
+Function that takes a Lexical editor config and modifies it to add the necessary
+`nodes` and `theme` to make [`LiveblocksPlugin`][] works correctly.
+
+```tsx
+import { liveblocksConfig } from "@liveblocks/react-lexical";
+
+const initialConfig = liveblocksConfig({
+  namespace: "MyEditor",
+  theme: {},
+  nodes: [],
+  onError: (err) => console.error(err),
+});
+```
+
+The config created by `liveblocksConfig` should be passed to `initialConfig` in
+[`LexicalComposer`][].
+
+```tsx highlight="7-12,16"
+import { LexicalComposer } from "@lexical/react/LexicalComposer";
+import { RichTextPlugin } from "@lexical/react/LexicalRichTextPlugin";
+import { ContentEditable } from "@lexical/react/LexicalContentEditable";
+import { LexicalErrorBoundary } from "@lexical/react/LexicalErrorBoundary";
+import { liveblocksConfig, LiveblocksPlugin } from "@liveblocks/react-lexical";
+
+const initialConfig = liveblocksConfig({
+  namespace: "MyEditor",
+  theme: {},
+  nodes: [],
+  onError: (err) => console.error(err),
+});
+
+function Editor() {
+  return (
+    <LexicalComposer initialConfig={initialConfig}>
+      <LiveblocksPlugin />
+      <RichTextPlugin
+        contentEditable={<ContentEditable />}
+        placeholder={<div>Enter some text...</div>}
+        ErrorBoundary={LexicalErrorBoundary}
+      />
+    </LexicalComposer>
+  );
+}
+```
+
+Note that `liveblocksConfig` sets `editorState` to `null` because
+`LiveblocksPlugin` is responsible for initializing it on the server.
+
+## useEditorStatus
+
+Returns the current editor status.
+
+```ts
+import { useEditorStatus } from "@liveblocks/react-lexical";
+
+const status = useEditorStatus();
+```
+
+The possible value are:
+
+- `not-loaded`: Initial editor state when entering the room.
+- `loading`: Once the editor state has been requested by `LiveblocksPlugin`.
+- `synchronizing`: Not working yet at runtime! Will be available in a future
+  release.
+- `synchronized`: The editor state is sync with Liveblocks servers.
+
+## FloatingComposer
+
+Displays a `Composer` near the current lexical selection.
+
+```tsx highlight="3"
+<LexicalComposer initialConfig={initialConfig}>
+  <LiveblocksPlugin>
+    <FloatingComposer />
+  </LiveblocksPlugin>
+</LexicalComposer>
+```
+
+{/* TODO: Image */}
+
+Submitting a comment will attach an annotation thread at the current selection.
+Should be nested inside [`LiveblocksPlugin`][].
```

Comment:
`<FloatingComposer />` does not currently need to be placed inside `LiveblocksPlugin` component. I wonder if we should still encourage this just to be safe.